src/breeze.cr

```diff
@@ -23,5 +23,6 @@ module Breeze
   Habitat.create do
     setting database : Avram::Database.class, example: "AppDatabase"
     setting enabled : Bool, example: "Lucky::Env.development?"
+    setting skip_breeze_if : Proc(HTTP::Server::Context, Bool)?
```

Comment:
The setting in the LogHandler is just `skip_if`, should we just copy that? I wasn't sure if it was clear what you were actually skipping... 🤔

Reply:
Could there be a place to add habitat settings for the request specific part of breeze? `skip_breeze_if` doesn't say anything about being request specific other than the type signature

Reply:
So you mean a separate set of Habitat settings? Like `BreezeAction.configure` ?

Reply:
Or I could call this `skip_breeze_pipes_if`....`skip_request_if` `skip_breeze_request_if` I'm open to names on it.

Reply:
If it makes sense, I'd like to have `Breeze.configure` be for shard level configuration and then have `Breeze::Request.configure`, `Breeze::Database.configure`, etc for configuring the various pieces. Does that make sense? Those are just random made up names

Reply:
Oh, I see what you're saying. Interesting 🤔  I think I can get behind that. The installer will set all that up for you anyway.  Want me to throw all that in to this PR and give it a shot? Or do a separate one?

Reply:
You can use this just to see if it works